patches/chrome-test-android-browsertests_apk-AndroidManifest.xml.jinja2.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/test/android/browsertests_apk/AndroidManifest.xml.jinja2 b/chrome/test/android/browsertests_apk/AndroidManifest.xml.jinja2
+index 6912b0853fec2..979650ebd1914 100644
+--- a/chrome/test/android/browsertests_apk/AndroidManifest.xml.jinja2
++++ b/chrome/test/android/browsertests_apk/AndroidManifest.xml.jinja2
+@@ -137,6 +137,7 @@
+                  android:isolatedProcess="false"
+                  android:exported="false" />
+         {% endfor %}
++        {% include "//brave/android/java/AndroidManifest.xml" %}
```

Comment:
do not patch here, we can use our own file here https://github.com/brave/brave-core/pull/8962/files#diff-02a56120c401438355c06b792cc494b41c2440cab96b107ce8fc8a49353b22f4R966